VLC for iOSUITests/TestHelper.swift

```diff
@@ -0,0 +1,58 @@
+/*****************************************************************************
+ * TestHelper.swift
+ * VLC for iOSUITests
+ *****************************************************************************
+ * Copyright (c) 2018 VideoLAN. All rights reserved.
+ * $Id$
+ *
+ * Authors: Mike JS. Choi <mkchoi212 # icloud.com>
+ *
+ * Refer to the COPYING file of the official project for license.
+ *****************************************************************************/
+
+import Foundation
+import XCTest
+
+struct TestHelper {
+    let localizationBundle: Bundle
```

Comment:
something that I didn't think about in my earlier review but that a friend pointed out is to use accessibility element identifiers. They don’t need to be localized! Since we need to make the app accessibility friendly anyway this will kill two birds with one stone :)

Reply:
So it looks like none of the elements in the app has its `accessibilityIdentifier` set. How should we go about this? Update every single element's `accessibilityIdentifier`?

Reply:
yes exactly. It needs to be added